src/screens/Order/OrderUIUtils.res

```diff
@@ -472,3 +474,72 @@ let getOrdersList = async (
 let isNonEmptyValue = value => {
   value->Option.getOr(Dict.make())->Dict.toArray->Array.length > 0
 }
+
+let viewsArray: array<viewTypes> = [All, Succeeded, Failed, Dropoffs, Cancelled]
```

Comment:
Can you provide polymorphic variants so you won't be needing mapper check APIUtils file for reference